notebooks/matrix_tube_pipeline_seqcount_norm.ipynb

```diff
@@ -804,33 +1046,41 @@
     "            dest_wells = np.array(plate_df[well_col]),\n",
     "            sample_names = np.array(plate_df[PM_SAMPLE_KEY]),\n",
     "            sample_plates = np.repeat(syndna_plate,plate_df.shape[0]))"
-   ],
-   "outputs": [],
-   "execution_count": null
+   ]
   },
   {
    "cell_type": "code",
-   "metadata": {},
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2025-03-04T00:18:38.967908Z",
+     "start_time": "2025-03-04T00:18:38.964544Z"
+    }
+   },
+   "outputs": [],
    "source": [
     "if is_absquant(plate_df):\n",
     "    if os.path.isfile(syndna_picklist_fp):\n",
     "        print(\"Warning! This file exists already.\")"
-   ],
-   "outputs": [],
-   "execution_count": null
+   ]
   },
   {
    "cell_type": "code",
-   "metadata": {},
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2025-03-04T00:18:40.110125Z",
+     "start_time": "2025-03-04T00:18:39.965204Z"
+    }
+   },
+   "outputs": [],
    "source": [
     "if is_absquant(plate_df):\n",
     "    with open(syndna_picklist_fp, 'w') as f:\n",
     "        f.write(syndna_picklist)\n",
```

Comment:
Minor but this looks to be lacking a trailing newline on write and in the string being written.

Reply:
Hmm, if those are necessary, I am not sure why their lack hasn't been complained about by the lab.  I will note that the syndna picklist generation needs to be rewritten (see https://github.com/biocore/metagenomics_pooling_notebook/issues/270 ) so my preference (since the newline situation doesn't seem to be actively distressing any users) would be to handle it at the same time as that fix.